### PR TITLE
Bugfix / equipment caching and highlighting

### PR DESCRIPTION
**Pull request by opyh**
- fixes several bugs in marker highlighting
- fixes non-highlighted equipment markers and non-highlighted associated equipment
- minimally changes how elevator/escalator icons look
- comes with a few code cleanups

### CODE REVIEW

File: src/components/NodeToolbar/Equipment/EquipmentItem.js [hunk 1]
```diff
@@ -197,7 +197,7 @@ const StyledEquipmentItem = styled(EquipmentItem)`
     &:last-of-type {
       margin-right: 0.75em;
     }
-    border: 1px solid ${colors.borderColor};
+    border: 1px solid ${colors.borderColor || 'gray'};
```
Comment (mutaphysis) [inline src/components/NodeToolbar/Equipment/EquipmentItem.js]: When does this happen?

File: src/App.js [hunk 2]
```diff
@@ -887,7 +888,7 @@ class App extends React.Component<Props, State> {
   onEquipmentSelected = (placeInfoId: string, equipmentInfo: EquipmentInfo) => {
     this.props.routerHistory.replace('equipment', {
       id: placeInfoId,
-      eid: equipmentInfo._id,
+      eid: get(equipmentInfo, 'properties._id'),
```
Comment (mutaphysis) [inline src/App.js]: 0: In EquipmentInfo.js:91 we still say that there is an _id on the top level, maybe remove that?

File: src/components/NodeToolbar/Equipment/EquipmentItem.js (hunk 1 above)
Comment (opyh) [reply src/components/NodeToolbar/Equipment/EquipmentItem.js]: This is to make FlowType happy.

File: src/App.js (hunk 2 above)
Comment (opyh) [reply src/App.js]: Yup! :)

Comment (opyh) [reply src/App.js]: https://trello.com/c/rZ3BYj1K/144-fix-equipmentinfo-type-id-in-wheelmap-react-frontend

**Pull request merged**